analyses/molecular-subtyping-HGG/07-HGG-molecular-subtyping-combine-table.Rmd

```diff
@@ -95,7 +95,23 @@ dna_df <- mutation_df %>%
 ```
 
 In earlier work on this module, we observed duplicated `sample_id`.
-We removed non-tumor samples (e.g., derived cell lines), does this issue persist?
+Yes
+
+```{r}
+histologies_df %>%
+  filter(sample_id %in% dna_df$sample_id[duplicated(dna_df$sample_id)],
+         experimental_strategy %in% c("WGS"),
+         sample_type == "Tumor") %>%
+  group_by(sample_id) %>%
+  arrange(sample_id) %>%
+  dplyr::select(Kids_First_Biospecimen_ID,sample_id,composition) %>%
+  summarize(
+            composition =  paste(composition, collapse = ", "))
+
+```
+The duplicates are mostly from multiple Solid tumor samples per sample_id or there are non-tumor ( derived cell-line ) samples for sample_id 
+
+
```

Comment:
Given the update to include derived cell lines, I'm not sure we need this section any more. As a reader, it's not clear to me how the 

> The duplicates are mostly from multiple Solid tumor samples per sample_id 

Gets resolved or handled. I understand the derived cell lines part.

Reply:
In this chunk I only wanted to show which sample ids have multiples and if they are Solid tumors or cell-lines. 

I can specify that we are only resolving the cell-lines using the new file, we aren't resolving the multiple Solid tumor sample id issue.